cmd/uplink/cmd/setup_test.go

```diff
@@ -115,3 +120,59 @@ func TestDefaultPortAppliedToSatelliteAddrWithPortColonButNoPort(t *testing.T) {
 	assert.Equal(t, "ahost:7777", setupCmd.Flags().Lookup("satellite-addr").Value.String(),
 		"satellite-addr should contain default port when no port specified")
 }
+
+func TestSaveEncriptionKey(t *testing.T) {
+	expKey := make([]byte, rand.Intn(20)+1)
+	_, err := rand.Read(expKey)
+	require.NoError(t, err)
+
+	t.Run("ok", func(t *testing.T) {
+		dir, err := ioutil.TempDir("", "storj-test-cmd-uplink")
+		require.NoError(t, err)
+		defer func() { _ = os.RemoveAll(dir) }()
+
+		fname := filepath.Join(dir, ".enc.key")
+		err = cmd.SaveEncryptionKey(expKey, fname)
+		require.NoError(t, err)
+
+		// Read the key from the file to compare that it matches with the saved one.
+		f, err := os.Open(fname)
+		require.NoError(t, err)
+		defer func() { _ = f.Close() }()
```

Comment:
It's a test, I don't think that report an error for this case is relevant for the test.

Reply:
We check all errors in tests too. So here you are supposed to have something like:
```go
defer func() { require.NoError(t, f.Close()) }()
```

Reply:
I knew that I could do so, but the reasons are the same that the ones commented in https://github.com/storj/storj/pull/1782#discussion_r278218338

I wait to reply on it, because it's the same case and with one thread is enough 😅